src/interfaces.ts

```diff
@@ -11,6 +10,11 @@ export namespace ISqlite {
     params?: any[]
   }
 
+  export interface SQLStatement {
+    sql: string
+    values?: any[]
```

Comment:
I matched the existing code pattern (i.e. no semicolons, using `any`), let me know if you prefer `unknown` instead.